milli/src/external_documents_ids.rs

```diff
@@ -112,6 +141,15 @@ impl Default for ExternalDocumentsIds<'static> {
     }
 }
 
+/// Returns the value of the `IndexedValue` with the highest _index_.
+fn indexed_last_value(indexed_values: &[IndexedValue]) -> Option<u64> {
+    indexed_values
+        .iter()
+        .copied()
+        .reduce(|a, b| cmp::max_by_key(a, b, |iv| iv.index))
```

Comment:
```suggestion
        .max_by_key(|iv| iv.index)
```
Can't we do that here?

Reply:
That's absolutely right, maybe I am redeclaring the internal algorithm inside `Iterator::max_by_key` :)